extensions-contrib/parquet-extensions/README.md

```diff
@@ -0,0 +1,420 @@
+          "fieldName": "cost_b_2nd_tag"
+        }
+      ],
+      "granularitySpec": {
+        "type": "uniform",
+        "segmentGranularity": "DAY",
+        "queryGranularity": "DAY",
+        "rollup": true,
+        "intervals": [
+          "2017-01-26T00:00:00.000Z/2017-01-27T00:59:59.000Z"
+        ]
+      }
+    },
+    "ioConfig": {
+      "type": "hadoop",
+      "inputSpec": {
+        "type": "static",
+        "inputFormat": "io.druid.data.input.parquet.DruidParquetInputFormat",
+        "paths": "example/events/events_with_all_datatypes.parquet"
+      },
+      "segmentOutputPath": "/tmp/segments"
+    },
+    "tuningConfig": {
+      "type": "hadoop",
+      "workingPath": "tmp/working_path",
+      "version": "2017-02-09T07:36:09.249Z",
+      "partitionsSpec": {
+        "type": "hashed",
+        "targetPartitionSize": 750000,
+        "maxPartitionSize": 750000,
+        "assumeGrouped": false,
+        "numShards": -1,
+        "partitionDimensions": [
+        ]
+      },
+      "shardSpecs": {
+      },
+      "indexSpec": {
+        "bitmap": {
+          "type": "concise"
+        },
+        "dimensionCompression": "lz4",
+        "metricCompression": "lz4",
+        "longEncoding": "longs"
+      },
+      "maxRowsInMemory": 75000,
+      "leaveIntermediate": true,
+      "cleanupOnFailure": true,
+      "overwriteFiles": false,
+      "ignoreInvalidRows": false,
+      "jobProperties": {
+        "hdp.version": "2.5.3.58-3",
+        "mapreduce.job.user.classpath.first": "true",
+        "mapreduce.task.timeout": "1800000",
+        "mapreduce.map.memory.mb": "2048",
+        "mapreduce.map.java.opts": "-server -Xmx3072m -Duser.timezone=UTC -Dfile.encoding=UTF-8",
+        "mapreduce.reduce.memory.mb": "2048",
+        "mapreduce.reduce.java.opts": "-server -Xmx8192m -Duser.timezone=UTC -Dfile.encoding=UTF-8",
+        "mapreduce.job.queuename": "default",
+        "mapred.child.ulimit": "16777216",
+        "mapreduce.map.output.compress": "true",
+        "mapred.map.output.compress.codec": "org.apache.hadoop.io.compress.GzipCodec"
+      },
+      "combineText": false,
+      "useCombiner": false,
+      "buildV9Directly": false,
+      "numBackgroundPersistThreads": 0,
+      "forceExtendableShardSpecs": false
+    },
+    "uniqueId": "715f31321096480b869aca2cc9c5d252"
+  },
+  "hadoopDependencyCoordinates": [
+    "org.apache.hadoop:hadoop-client:2.6.0"
+  ]
+}
+```
+
+ ##### ***Field***
+
+| Field	| Type       | Description           | Required  |
+| ------|:-------    |:-------------:        | :-----|
+| **key**|String | Column name of the field to be extracted | true |
+| **fieldType** | String [mapping is mentioned below] | Type of the field | true |
+|rootFieldName|String|This attribute is used while extracting a dimension or metric from a map|optional (`true` when the field is a map else none )|
+|index|Integer|Used while referencing a list by its index position|no|
```

Comment:
Can you mention here that the parser will build a multivalued dimension from a LIST when `index` is not specified?

Reply:
Sure, we will include that